### DISCUSSION TITLE
案例：《乘着公车出逃：美国如何安置无家可归者》

### DISCUSSION THREAD

**Discussion by Ceuniu** [data-journalism/data-journalism.github.io#58] [Show and tell]
![bussed_out](https://user-images.githubusercontent.com/543384/139793087-67a065ec-b5cc-4aa4-bebd-a898fe8da6e3.gif)

![bussed_out1](https://user-images.githubusercontent.com/543384/139793193-6b92e331-4dcf-4da9-a6c7-50fdb76e2994.gif)

![bussed_out2](https://user-images.githubusercontent.com/543384/139793201-f235efc2-f8ff-40a6-8761-1552ab6fcdd2.gif)

[Bussed out.docx](https://github.com/data-journalism/data-journalism.github.io/files/7453567/Bussed.out.docx)

**Comment by xueee-2001** [comment]
# Bussed out——How America moves its homeless

https://www.theguardian.com/us-news/ng-interactive/2017/dec/20/bussed-out-america-moves-homeless-people-country-study

> Each year, US cities give thousands of homeless people one-way bus tickets out of town. An 18-month nationwide investigation by the Guardian reveals, for the first time, what really happens at journey’s end

By the Outside in America team

20 December 2017

《乘着公车出逃：美国如何安置无家可归者》

- 凯度信息之美奖 2018年获奖作品 https://www.informationisbeautifulawards.com/
- 佛罗里达大学调查性数据新闻奖

1.	作品简介

1.1写作背景
2021年3月，美国房屋和城市发展部（HUD）发布了《2020年美国无家可归者年度评估报告》。据统计，美国大约有58万人无家可归，比2019年（56.77万）有所增加。全国无家可归者连续四年增加。

调查机构“经济圆桌会议”发布了一份研究报告认为，疫情下的失业潮导致无家可归者数量激增，而这一趋势将持续恶化至2023年。该报告预测，2023年全美无家可归者人数将达到惊人的116.8万，超过疫情前的2倍。

“无家可归者搬迁计划（homeless relocation programs）”——30多年来，美国大多数城市一直将公车项目作为解决无家可回的人员流向问题的廉价且快速的方法。政府通过公车项目提供单程票，将无家可归的人们送出城市。表面上，这一项目帮助他们去往想去的地方，让他们和家人或朋友住在一起，以获得一些社会帮助，解决生活困境，但长久以来很少有人对此计划的实际效果进行过评估。

1.2主要内容
在为期18个月的调查中，《卫报》对无家可归者的搬迁计划进行了详细分析。为了确定哪些城市有公车项目，《卫报》记者联系了美国25个大城市的相关工作人员，还查阅了20世纪80年代初至今的档案，最终收到了来自16个县、市的数据，创建了34240次搬迁行为的数据库，并分析了这些搬迁行动对城市和旅行者的影响。

除了数据，文章还有对多位搬迁计划的亲身参与者以及计划的相关负责人的采访，将采访内容与数据相结合，交织呈现了调查结果。

1.3文章结论
通过调查，《卫报》发现，尽管不少人通过这一项目改善了生活状况，但仍然存在诸多问题：有些无家可归者一旦坐上离开的公车，就终身被禁止回来；政府对迁移者的处境不闻不问，而很多迁移者之后过得很糟糕；一些被警方通缉的人甚至也得到了离开的车票；并未解决问题本质，而是各个州之间的踢皮球，用文中的话来说就是，这个项目“就像是在泰坦尼克号甲板的椅子上四处转移，而不是真正地减少无家可归者的数量”。

2.	可视化梳理分析
2.1可视化解读
第一张图片是一个流浪汉的路线图，展现了他从旧金山到印第安纳波利斯超过两千英里的搬迁旅程。作者用这张图作为故事的开始，引入了对无家可归者搬迁计划的介绍。

<img width="416" alt="image" src="https://user-images.githubusercontent.com/74866145/139685250-d979c044-0998-475e-ac07-dad6da7d09bc.png">

第二张图也是在地图的基础上绘制的，颜色越深代表无家可归者比例越高。无家可归的流浪者规模巨大，西部地区尤甚，加利福尼亚、内华达、俄勒冈和华盛顿等州无家可归率最高。

<img width="416" alt="image" src="https://user-images.githubusercontent.com/74866145/139685349-39fc9fc9-6101-446f-a39e-d8663ad089fe.png">

随后，《卫报》收集了来自16个城市和县的无家可归者的旅程数据，制作成了无家可归者人口流动图。这个动态气泡图在地图上呈现了2011年到2017年超过20000名无家可归者在美国各地的流动状态，红点表示城市，蓝色圆圈的大小则表示了吸纳流动人口的规模。我们可以看到大多数人西部地区的人向东部迁移，大多数东部的人则向东部其他地区和中部迁移。

<img width="416" alt="image" src="https://user-images.githubusercontent.com/74866145/139685382-39e4802b-8405-4cdb-9a14-fb9cefdf74de.png">
 
这一系列图均是以纽约市为起点，以旅程的距离绘制抛物线，第一张图显示了有650人飞往国外，第二张图特别标注了接收最多纽约无家可归者的城市波多黎各，第三张图则呈现了美国大陆最受欢迎的城市。这些图直观地表现了无家可归者搬迁规模浩大，旅途的距离非常遥远。
 
<img width="388" alt="image" src="https://user-images.githubusercontent.com/74866145/139685411-9510ab8a-161b-4f1c-9355-dc2aca963787.png">
<img width="430" alt="image" src="https://user-images.githubusercontent.com/74866145/139685431-f641aee8-6701-499e-8b31-d12010717c61.png">
<img width="415" alt="image" src="https://user-images.githubusercontent.com/74866145/139685447-12d3f823-a66e-4420-8da7-71d9031c15c0.png">

作者还绘制了一张搬迁前后城市收入中位数的分流图。红色部分是指流浪者去往了收入中位数更高的城市，这部分人口仅占12%。88%的人搬迁至经济更不景气的地区，这一方面是因为可以获得更低的住房和生活成本，在另一方面却也体现了高收入城市的排外性。正如其中一名信源在报道中所说的，无家可归者搬迁计划将他们转移到其他陷入困境的地区，却忽视了根本问题的解决。
<img width="248" alt="image" src="https://user-images.githubusercontent.com/74866145/139685481-6ea33f51-cd24-4401-9bc6-4ea65bb63f8d.png">

这幅图揭示了到底是哪些人接收了无家可归者，从父母子女、兄弟姐妹到朋友和雇主，种类非常多。每个同心圆中粒子的数目代表了规模，越靠近圆心，对应的人或机构占据的比例就越高。
 
<img width="376" alt="image" src="https://user-images.githubusercontent.com/74866145/139685506-a173a89d-bbe3-4620-b077-8dcf2c159ca6.png">

最后，作者以旧金山市为例，绘制了三幅动态的图表。这些图表涉及的时间段均是2005年到2017年，用粒子都代表着无家可归的人，粒子不断变化的位置和不断增加的总数量代表了人员的流动和增加。在12年间，共有10570人通过搬迁计划离开旧金山，这也意味着，如果旧金山市从来没有施行这个计划，到2017年旧金山市将有18070名流浪者。
  
<img width="319" alt="image" src="https://user-images.githubusercontent.com/74866145/139685518-674edea4-1a65-4b8b-85df-71b69fc3a2d3.png">
<img width="300" alt="image" src="https://user-images.githubusercontent.com/74866145/139685530-d89eb803-1f5d-4cfc-b3a8-279ea1abbad5.png">
<img width="282" alt="image" src="https://user-images.githubusercontent.com/74866145/139685544-dfdcadf5-91b0-4052-a8c4-1ff6af185fa5.png">

2.2可视化评价
（一）优点：
1.图表类型丰富，具有创新性。作者创造了多种形式，如用同心圆表示出不同关系网络对无家可归者的接纳数目，而读者可以发现越靠近圆心，血缘关系就越强，所以接纳者类型在数量上的关系也和他们与流浪者血缘关系联系紧密。他们还用抛物线图表示出流浪者迁移的距离远近，而点在抛物线上的轨迹与流浪者从一个地方被丢向别处的生活境遇不乏相似之处。
2.动画效果出色，生动表现数据的时间变化、数量变化和距离远近。创作者自述使用了Greensock工具集，这个工具集使得作者能够自由控制每一个粒子的运动，从而将读者的注意力吸引到关键数据上或者动态数据本身。如动态气泡图中数以万计的粒子从16个城市向目的地发射，读者可以很明显地感觉出东部人口的聚集。
（二）缺点：
这篇文章采用了“scrollytelling”的方式，也就是运用了此前课上提到的Scrollama脚本。与其他可视化作品不同的是，它的图表是相对静止的，随着鼠标滚轮的转动，是文字而非图片在屏幕上移动。这也造成了一个问题，即部分文字在滑动过程中遮盖了图表，这时候图文就难以辨别，这是这篇可视化作品在视觉呈现上应该改进的地方。

3.技术实现
3.1数据来源和处理
①从美国最大的25个城市中确定入选城市，并联系入选城市的无家可归者、住房部门或警察部门，请求获取无家可归者个人旅程目的地、日期等基本信息。
②拨打获取的电话号码，采访买过公交车票的人。对于没有号码的700名，在Nexis数据库中搜索联系方式，并在社交媒体上发送消息。此外，纽约、洛杉矶、波特兰、基韦斯特和圣胡安的记者访问了收容所和无家可归者营地。
③ 根据公共记录法，制作团队从16个城市和县获得了安置无家可归者的数据。这些城市和县为无家可归者提供免费巴士票以让他们住在其他地方。使用美国城市数据库根据人口普查数据将每张公交车票与每一段旅途及迁移结果进行匹配。如果出现不匹配项则修改或删除这一项。团队总共整理了 34,000 人的旅行信息，并进一步分析这些数据以了解年龄、性别、种族和目的地的趋势，以及随时间的变化，这取决于可用的变量。
最终，在美国地图的可视化呈现部分，报道选择分析了2011-2016年在美国本土进行的 21,000 次旅行。为了得出人均无家可归率，使用住房和城市发展部发布的 2016 年州无家可归人口估计数和美国人口普查局 2016 年的州人口数据。
 
<img width="240" alt="image" src="https://user-images.githubusercontent.com/74866145/139685568-9a5de331-6fb2-4840-98a0-0e647b18266c.png">

④ 从纽约迁往外地的可视化图中，使用Google Maps APIs（通过R中的ggmap工具包）获取来自纽约的航班信息，将地理位置附加到来自纽约数据集的所有非美国旅行。按国家/地区对旅行者进行分组，并使用每个国家/地区内目的地的平均地理位置，计算出与纽约的距离。为了显示来源城市和目的地城市平均收入的差异，首先需要得出这些城市的平均收入。制作团队使用了US Census Factfinder并将检索结果与美国城市数据库交叉参考，以确保收入数据与正确的城市相匹配。
通过R对数据进行清理后，得到以下几个csv文件：us_destination_city_totals.csv（所有目的地城市），us_bus_trips_sample.csv（公共汽车旅行样本），income_origin_destination_diff.csv（来源地与目的地收入差异）。
 
<img width="388" alt="image" src="https://user-images.githubusercontent.com/74866145/139685582-fe505ed0-2c23-42d6-83db-28ab261e04e6.png">
<img width="286" alt="image" src="https://user-images.githubusercontent.com/74866145/139685600-7bdf7bc8-5f40-431c-a8e7-94a50ce8bc62.png">

 
⑤ 谁在搬迁目的地收容无家可归者的数据十分有限，只有佛罗里达州的劳德代尔堡、萨拉索塔、西棕榈滩保持着记录。制作团队将这三个城市的数据汇总，得出接收无家可归旅客的人和机构分别是哪些。
 
<img width="268" alt="image" src="https://user-images.githubusercontent.com/74866145/139685624-8753a146-a49b-40ae-a295-6a1be52c0ef3.png">

⑥ 旧金山市2005年开启处理无家可归者的计划，制作团队根据计划的数据集展示了进出城市的无家可归者数量。（尽管这并不是美国所有bus programs的数据，但它可以让人们一定程度上了解bus program的影响。
 
<img width="277" alt="image" src="https://user-images.githubusercontent.com/74866145/139685638-c0f21495-b237-4c0b-8d33-ccebe5730bfd.png">


3.2可视化工具
工具：R、d3.js、canvas、Greensock
R用于从网站上抓取数据，并让csv或excel表格的数据格式更清晰。
d3.js用于呈现视觉效果，如迁移过程的动态图。
Canvas用于制作网页，形成完整的艺术作品。
使用Greensock的时间轴，让用户向下滚动屏幕时出现视觉元素的转换。

4.总结

4.1 意义
（1）回应现实，第一次对美国无家可归者安置计划进行详细分析。
（2）利用数据审视这一安置计划的合理性。在项目的设计初衷中，没有人会在到达另一个地方后仍然无家可归。但现实的结果并不是如此。数据和采访揭示了一部分原因。 
①无家可归者：名为“回家”和“家庭团聚”的重新安置项目的基本假设是，回到家乡或亲戚身边将带来一个康复、改善的过程。但对一些人来说，无家可归的原因是家庭冲突和关系破裂，这些问题可能根植于他们要返回的地方。
②接收无家可归者的人：即使是那些向工作人员保证他们可以照顾其亲人的无家可归者的亲属，也可能难以实现这一承诺。
③政府/机构：缺乏足够的资金和人力对后续情况进行跟踪。
（3）提供了一个范本：如何在数据有限的情况下尽可能地利用已有数据完成报道，或是尽可能地去寻找到可用数据。
（4）拓展新闻的形式：在文本外，结合使用数据可视化、视频、摄影，更为鲜活地向读者解释了无家可归者在美国各地流动的影响。

4.2 可改进的部分
（1）数据可视化 
①部分图表可以进一步增强交互性，例如这张图可以设置为点击即可查看每一个州无家可归的人数，让读者更加清晰地了解到每个州无家可归者的情况。
<img width="415" alt="image" src="https://user-images.githubusercontent.com/74866145/139685665-067bfb43-5f16-4071-9ac3-cb7c47b040f0.png">
②这篇文章采用了“scroll-telling”的方式，也就是运用了此前课上提到的Scrollama脚本。但由于这篇文章的图表相对静止，随着鼠标滚轮的转动，是文字而非图片在屏幕上移动。这也造成了一个问题，即部分文字在滑动过程中遮盖了图表，这时候图文就难以辨别。作者可以考虑如何将图表和页面的设计更好地结合起来，便于读者阅读。

（2）文章内容 
①就文章开头的三个问题而言（这些⼈被搬到哪⾥去了？这些项⽬对发送和接收的城市有什么影响？这些⽆家可归的⼈到达⽬的地后会发⽣什么？），这篇报道似乎没有很好地回答集中接收这些无家可归者的地区受到了哪些影响。
②文中的数据显示，政府或相关机构倾向于将无家可归的人送到经济水平更差的地区。然而受访者表示，那些经常陷入无家可归的人也许本来就来自几十年来经历过大衰退的社区。因此把他们转移到其他贫困的社区，无法从根本上解决这一问题。这里或许可以利用数据统计这些无家可归者的出生地、性别、年龄、种族等信息，更有力地说明这一情况，而不是仅仅通过一个单独信源来解释。

**Comment by chengjun** [comment]
LexisNexis是世界著名的数据库，全球许多著名法学院、法律事务所、高科技公司的法务部门都在使用该数据库。该数据库连结至40亿个文件、11,439个数据库以及36,000个来源，资料每日更新。

**Reply by chengjun** [reply]
American FactFinder has been decommissioned and is no longer available. Data are now available at:https://data.census.gov/cedsci/
